runtime/frequency/src/lib.rs

```diff
@@ -1119,6 +1114,12 @@ impl pallet_frequency_tx_payment::Config for Runtime {
 	type MaximumCapacityBatchLength = MaximumCapacityBatchLength;
 	type MsaKeyProvider = Msa;
 	type MsaCallFilter = MsaCallFilter;
+	#[cfg(feature = "runtime-benchmarks")]
+	type MsaBenchmarkHelper = Msa;
```

Comment:
I believe two of these can be removed. However, if that is not the case, I’d suggest creating a single BenchmarkHelper trait that includes all the necessary functions. This would simplify the dependencies and make the wiring much easier and cleaner.